Kernel/OrbitalFrequencies.m

```diff
@@ -127,44 +127,133 @@
 
 ]
 
+(*Simplified math from Fujita and Hikida in a way that still evaluates when x = 0 -> zm = 1.*)
+KerrGeoMinoFrequencies[a_, p_, e_, x_] := 
+ Module[{M = 1, En, L, Q, r1, r2, r3, r4, z1, z2, kr, k\[Theta], rp, 
+   rm, hr, hp, hm, Kkr, Kk\[Theta], Ekr,
+   Ek\[Theta], Pihr, Pihp, Pihm, 
+   Piz1, \[CapitalUpsilon]r, \[CapitalUpsilon]\[Theta], \
+\[CapitalUpsilon]\[Phi], \[CapitalGamma], \[CapitalOmega]r, \
+\[CapitalOmega]\[Theta], \[CapitalOmega]\[Phi]}(*\[Epsilon]0,zm,
+  a2zp,\[Epsilon]0zp,zmOverZp,kr,
+  k\[Theta],\[CapitalUpsilon]r,\[CapitalUpsilon]\[Theta],rp,rm,hr,hp,
+  hm,\[CapitalUpsilon]\[Phi],\[CapitalGamma]}*), {En, L, Q} = 
+   Values[KerrGeoConstantsOfMotion[a, p, e, x]];
+  {r1, r2, r3, r4} = KerrGeoRadialRoots[a, p, e, x, En, Q];
+  (*{zminus,zplus}={Sqrt[1-x^2],Sqrt[Q/(1-x^2)]}(* = 
+  KerrGeoPolarRoots[a,p,e,x,En,L]*),
+  {zm,zp}={zminus^2,zplus^2},(*For some reason, 
+  Fujita and Hikida define zm=zminus^2 \[Equal] z1 and zp=
+  Q/(L^2 \[Epsilon]0 zm), with \[Epsilon]0=a^2(1\[Minus]En^2)/L^2, 
+  so that zp \[Equal] zplus^2/(a^2(1-En^2)) \[Equal] z2/(a^2(1-
+  En^2)). *)
+  *)
+  z1 = 1 - x^2;
+  If [z1 == 0., z2 = a^2 (1 - En^2) + L^2;, z2 = Q/z1;];(*(a^2 (1-
```

Comment:
I think we'd prefer an assignment of the form `z2 = If[test, case1, case2];`.

Also, will this test work for exact 0 instead of floating point 0?

Reply:
```suggestion
  z2 = If [z1 == 0, a^2 (1 - En^2) + L^2, Q/z1];
```
Seems to work fine with either '0' or '0.'. 
I found that the syntax (0|0.) didn't work there, but it doesn't seem to be necessary.